modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,241 @@
+  maxReplicas: 3
+  metrics:
+  - resource:
+      name: cpu
+      target:
+        averageUtilization: 70
+        type: Utilization
+    type: Resource
+  minReplicas: 1
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: my-app
+----
+
+Some points to consider:
+
+Scale up can occur at most every 180 seconds. Each period, up to 100% of the currently running replicas may be added until the maximum configured replicas is reached. For scaling up there is no stabilization window. When the metrics indicate that the target should be scaled up, the target is scaled up immediately.
+
+Scale down can occur at most every 15 seconds. Each Period, up to 100% of the currently running replicas may be removed which means the scaling target can be scaled down to the minimum allowed replicas. The number of replicas removed is based on the aggregated calculations over the past 300 seconds of the stabilization window.
+
+Min replicas +
+
+* The minimum number of replicas that would be guaranteed to run at any given point of time.
+* Scale down policy would never remove replicas below this number.
+
+Max replicas +
+
+* The maximum number of replicas that are capped, beyond which no more replicas can be added for scale up.
+* Scale up policy would never add replicas above this number.
+
+Enabling HPA can result in customers incurring additional flow usage when your application scales horizontally. To avoid overages from unpredicted scaling, configure the maximum configured replicas judiciously to stay within purchased flow limits. Track your incurred flow usage through xref:general::usage-reports.adoc[usage reports].
+
+
+== Performance Considerations and Limitations
+
+For a successful horizontal autoscaling of your Mule apps, review the following performance considerations:
+
+* In Runtime Fabric, the policy in use was benchmarked for Mule apps with CPU Reserved: 0.45vCpu and Limit: 0.55vCpu, which corresponds to these settings:
++
+----
+        resources:
+          limits:
+            cpu: 550m
+          requests:
+            cpu: 450m
+----
++
+
+* Mule apps that scale based on CPU usage are a good fit with CPU based HPA. For example: +
+
+** HTTP/HTTPS apps with async requests
+** Reverse proxies
+** Low latency + high throughput apps
+** Dataweave transformations
+** APIKit Routing
+** API Gateways with policies
+
+* Non-reentrant apps that do not have built in parallel processing such as batch jobs, scheduler apps without re-entrancy and duplicate scheduling across apps and low throughput, high latency apps with large requests may not be a good fit with CPU based HPA.
+
+* Scale up and scale down performance can vary based on the replica size and the application type.
+
+* The policy is optimized for replica sizes with <850m. Larger replica sizes might take longer to scale.
+
+=== Limitations
+
+There are some limitations to consider:
+
+* Horizontal autoscaling does not work with clustering and rate limiting.
```

Comment:
@kevintroller - this limitation has been removed now, can you please confirm with @SparshAgarwal